docs(website): upgrade instruction (#872)
glasskube/glasskube: website/src/partials/_upgrade.mdx

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

<Tabs groupId="operating-systems">
  <TabItem value="mac" label="macOS">

>     On macOS, you can use [Homebrew](https://brew.sh/) to install and update Glasskube.
>     ```shell
>     brew install glasskube/tap/glasskube
>     ```
> 

Reviewer: ```suggestion
```

I suggest removing this section as we are in the upgrade section

    To upgrade Glasskube, use the following command:
    ```shell
    brew upgrade glasskube
    ```
  </TabItem>
  <TabItem value="linux" label="Linux">

>     You can install Glasskube using one of the package managers below.

Reviewer: ```suggestion
    You can upgrade Glasskube by reinstalling the latest package using one of the package managers below.
```

      ```
      {/* x-release-please-end */}
    </details>

    <details>
      <summary>DEB-based installation (Ubuntu/Debian)</summary>

      {/* x-release-please-start-version */}
      ```
      curl -LO https://releases.dl.glasskube.dev/glasskube_v0.10.1_amd64.deb
      sudo dpkg -i glasskube_v0.10.1_amd64.deb
      ```
      {/* x-release-please-end */}
    </details>

    <details>
      <summary>APK-based installation (Alpine)</summary>
      {/* x-release-please-start-version */}
      ```
      curl -LO https://releases.dl.glasskube.dev/glasskube_v0.10.1_amd64.apk
      apk add glasskube.apk
      ```
      {/* x-release-please-end */}
    </details>

    If you are using a distribution that does not use one of the package managers above, or require a 32-bit binary,
    check out additional download options attached to our [latest release](https://github.com/glasskube/glasskube/releases/latest).

    To upgrade Glasskube:
    1. Download the latest Glasskube binary from the [releases page](https://github.com/glasskube/glasskube/releases).
    2. Replace the existing binary with the new one.
  </TabItem>

  <TabItem value="win" label="Windows">

>     {/* x-release-please-start-version */}
>     Download the [windows archive](https://releases.dl.glasskube.dev/glasskube_v0.10.1_windows_x86_64.zip) from our
>     latest [Release](https://github.com/glasskube/glasskube/releases/latest) and unpack it using Windows Explorer.
>     {/* x-release-please-end */}
> 
>     To upgrade Glasskube:
>     1. Download the latest Glasskube executable from the [releases page](https://github.com/glasskube/glasskube/releases).
>     2. Replace the existing executable with the new one.

Reviewer: ```suggestion
    {/* x-release-please-start-version */}
    Download the [windows archive](https://releases.dl.glasskube.dev/glasskube_v0.10.1_windows_x86_64.zip) from our
    latest [Release](https://github.com/glasskube/glasskube/releases/latest), unpack it using Windows Explorer and replace the existing executable with the new one.
    {/* x-release-please-end */}
```
